docs/ref/templates/builtins.txt

```diff
@@ -1501,6 +1501,11 @@ For example::
 
 If ``value`` is ``None``, the output will be the string ``"nothing"``.
 
+.. versionchanged:: 2.0
```

Comment:
Given that the behavior of undefined variables isn't documented, I don't see a need for this note unless you also add that documentation. The note should be self-contained as described in https://docs.djangoproject.com/en/dev/internals/contributing/writing-documentation/#documenting-new-features.